### PR TITLE
Add CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by srbcheema1**
This PR is to add `CONTRIBUTING.md` file to help new-comers to make a PR and contribute in this repository. Fixes the issue  https://github.com/EBIvariation/vcf-validator/issues/157

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+We welcome community contributions to vcf-validator! To build tool locally, follow the instructions in the [README.md file](https://github.com/EBIvariation/vcf-validator/blob/master/README.md).
+
+
+## First Timers Welcome!
+
+New to open source? Here are a selection of issues we've made especially for first-timers. follow this [link](https://github.com/EBIvariation/vcf-validator/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner)
+
+
+## Making a New PR
+Once you found an interesting issue to work on you may check if someone other is not working on the same issue. you can ask once in the issue to get that issue assigned to you. once you get an approval go for the PR. While making a new PR do follow these norms:
```
Comment (jmmut) [inline CONTRIBUTING.md]: > ... issue. you...
After dot there should be an uppercase letter always :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- Tests - Add tests if new functionality is added to ensure that this functionality does not break in future.
+- Consistent Naming Scheme - Follow a consistent naming scheme. we prefer `snake_case`(underscored) instead of `camelCase`. naming of variables should be done in well explanatory way.
+- DRY Principle - Every piece of knowledge must have a single, unambiguous, authoritative representation within a system. Aslo called [Don't repeat yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
+- Avoid Deep Nesting - Too many levels of nesting make code harder to read.
+- Limit Line Length - It is a good practice to avoid writing horizontally long lines of code. There is no such limit but still having max lenght till 80 is good practive. For more follow this [link](https://stackoverflow.com/a/111009/6799074).
+- Bracket Style - We generally use attached brackets for `conditions` and `loops` and broken brackets for `functions` and `classes`. This style is known as [K&R style](https://en.wikipedia.org/wiki/Indentation_style#K&R).
```
Comment (jmmut) [inline CONTRIBUTING.md]: K&R, but we force always to use brackets.

Comment (srbcheema1) [reply CONTRIBUTING.md]: Ohh I see I should be more precise here. it should have been https://en.wikipedia.org/wiki/Indentation_style#Variant:_Linux_kernel

Comment (srbcheema1) [reply CONTRIBUTING.md]: it too is not using brackets in while loop. I think there is no name to the standard we use

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,32 +6,32 @@ We welcome community contributions to vcf-validator! To build tool locally, foll
 
 ## First Timers Welcome!
 
-New to open source? Here are a selection of issues we've made especially for first-timers. follow this [link](https://github.com/EBIvariation/vcf-validator/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner)
+New to open source? Here are a selection of issues we've made especially for first-timers. follow this [link](https://github.com/EBIvariation/vcf-validator/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner).
```
Comment (jmmut) [inline CONTRIBUTING.md]: put "follow" in uppercase

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -6,32 +6,32 @@ We welcome community contributions to vcf-validator! To build tool locally, foll
...
-- Consistent Naming Scheme - follow a consistent naming scheme. we prefer `snake_case`(underscored) instead of `camelCase`. naming of variables should be done in well explanatory way.
-- DRY Principle - Every piece of knowledge must have a single, unambiguous, authoritative representation within a system.
-- Avoid deep nesting - too many levels of nesting make code harder to read.
-- Limit line length - It is a good practice to avoid writing horizontally long lines of code.
-- Bracket style - we generally use attached brackets for `conditions` and `loops` and broken brackets for `functions` and `classes`.
+- Consistent Indentation - We generally prefer 4 spaces for indentation.
+- Comments & Documentation - Provide comments whenever required, so that it may be easy for other developers to read your code. Also avoid obvious comments.
+- Tests - Add tests if new functionality is added to ensure that this functionality does not break in future.
+- Consistent Naming Scheme - Follow a consistent naming scheme. we prefer `snake_case`(underscored) instead of `camelCase`. naming of variables should be done in well explanatory way.
```
Comment (jmmut) [inline CONTRIBUTING.md]: put "we" and "naming" in uppercase

Comment (srbcheema1) [reply CONTRIBUTING.md]: I think I strongly need to write a script to detect a lowercase letter after a `dot`. :sweat_smile:

**Pull request merged**